connectivity/check/check.go

```diff
@@ -239,6 +249,15 @@ type TestContext interface {
 	// EchoServices returns a map of all deployed echo services
 	EchoServices() map[string]ServiceContext
 
+	// ApplyPolicyYaml applies the given policy to the test context, returns the number of failures
+	ApplyPolicyYaml(ctx context.Context, yaml string) int
```

Comment:
Nit: use all capitals for acronyms

```suggestion
	// ApplyPolicyYAML applies the given policy to the test context, returns the number of failures
	ApplyPolicyYAML(ctx context.Context, yaml string) int
```

Reply:
Fixing, thanks!